scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -1251,6 +1276,14 @@ public Void doInTransaction(Session session) {
         });
     }
 
+    private int updateTasksStatuses(final Set<TaskId> taskIds, final TaskStatus taskStatus, Session session) {
```

Comment:
Same here isn't it better to have updateTasksStatus because each task has only one status.